packages/hebao_v1/contracts/iface/Wallet.sol

```diff
@@ -18,6 +18,8 @@ interface Wallet
 
     function owner() external view returns (address);
 
+    function previousOwner(uint idx) external view returns (address, uint);
```

Comment:
I'm not happy with this API, but I don't know how to come up with a more appropriate one.